libnavigation-ui/src/main/java/com/mapbox/navigation/ui/feedback/FeedbackBottomSheet.java

```diff
@@ -307,253 +280,118 @@ public void onFinish() {
-      R.string.mapbox_feedback_description_lane_guidance_incorrect));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.ROAD_KNOW_BY_DIFFERENT_NAME,
-      R.string.mapbox_feedback_description_road_known_by_different_name));
-
-    return list;
-  }
-
-  @NonNull
-  private List<FeedbackSubTypeItem> subTypeOfIncorrectAudioGuidance() {
-    List<FeedbackSubTypeItem> list = new ArrayList<>();
-
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.GUIDANCE_TOO_EARLY,
-      R.string.mapbox_feedback_description_guidance_too_early));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.GUIDANCE_TOO_LATE,
-      R.string.mapbox_feedback_description_guidance_too_late));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.PRONUNCIATION_INCORRECT,
-      R.string.mapbox_feedback_description_pronunciation_incorrect));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.ROAD_NAME_REPEATED,
-      R.string.mapbox_feedback_description_road_name_repeated));
-
-    return list;
-  }
-
-  @NonNull
-  private List<FeedbackSubTypeItem> subTypeOfRoutingError() {
-    List<FeedbackSubTypeItem> list = new ArrayList<>();
-
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.ROUTE_NOT_DRIVE_ABLE,
-      R.string.mapbox_feedback_description_route_not_drive_able));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.ROUTE_NOT_PREFERRED,
-      R.string.mapbox_feedback_description_route_not_preferred));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.ALTERNATIVE_ROUTE_NOT_EXPECTED,
-      R.string.mapbox_feedback_description_alternative_route_not_expected));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.ROUTE_INCLUDED_MISSING_ROADS,
-      R.string.mapbox_feedback_description_route_included_missing_roads));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.ROUTE_HAD_ROADS_TOO_NARROW_TO_PASS,
-      R.string.mapbox_feedback_description_route_had_roads_too_narrow_to_pass));
-
-    return list;
-  }
-
-  @NonNull
-  private List<FeedbackSubTypeItem> subTypeOfNotAllowed() {
-    List<FeedbackSubTypeItem> list = new ArrayList<>();
-
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.ROUTED_DOWN_A_ONE_WAY,
-      R.string.mapbox_feedback_description_routed_down_a_one_way));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.TURN_WAS_NOT_ALLOWED,
-      R.string.mapbox_feedback_description_turn_was_not_allowed));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.CARS_NOT_ALLOWED_ON_STREET,
-      R.string.mapbox_feedback_description_cars_not_allowed_on_street));
-    list.add(new FeedbackSubTypeItem(
-      FeedbackEvent.TURN_AT_INTERSECTION_WAS_UNPROTECTED,
-      R.string.mapbox_feedback_description_turn_at_intersection_was_unprotected));
+  };
 
-    return list;
+  @IntDef( {FEEDBACK_FLOW_IDLE, FEEDBACK_FLOW_SENT, FEEDBACK_FLOW_CANCEL})
+  @Retention(RetentionPolicy.SOURCE)
+  public @interface FeedbackFlowStatus {
```

Comment:
This interface is only used in the drop-in scenario. Could it be moved to a different package and made Kotlin internal (or Java package-private)?

Reply:
Good point!